src/containers/MasonryLayout.jsx

```diff
@@ -57,11 +57,16 @@ class MasonryLayout extends Component {
   render() {
     return (
       <div className='masonry'>
-        <div className='masonry-layout'>
-          {
-            this.createItems(this.state.items)
-          }
-        </div>
+        {
+          this.props.items.length > 0 ?
+            <div className='masonry-layout'>
+              {
```

Comment:
I have been pushing for a certain best practice across the team.

How about you evaluate this before the return function in the render to improve readability.

e.g

```
let itemHtml = "";
const {items} = this.state;
if(items.length){
itemHtml = items.map(...)
} else {
itemsHtml = "There are no activities at the moment"
}
```

And then pass the `itemHtml` within the return statement.

Reply:
@kimani-ndegwa I just pushed a new commit to fulfill line 2.
I'll be doing the condition check inside the return statement because I want to display two different elements. A `div` with className `masonry-layout` and a `h2` with className `items-unavailable-message`.